### ISSUE TITLE
[bug]: `aria-controls` on storybook navigation wrongfully used

### ISSUE THREAD

**Issue by leagrdv** [swisspost/design-system#5237]
### 🐞 Describe the bug

When the [Foundations > Typography > Overview](https://next.design-system.post.ch/?path=/docs/677cfabf-dbf0-4de2-ad07-6d5bfb9e2375--docs) navigation is opened, we can see that the parent `Typography` has an `aria-controls="foundations-typography-overview"` which should refer to the ID of the element it controls, but the child `Overview` does not have such ID.
This is also the case for `Foundations > Layout > *`.

### 📷 Screenshots

![Image](https://github.com/user-attachments/assets/a3df8b4c-add5-4032-91c9-a963c83422a2)
